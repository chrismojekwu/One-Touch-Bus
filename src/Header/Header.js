import React from 'react'
import logo from '../img/OTB.png'

export default function Header(){
    return (
        <img className="logo" src={logo}/>
    )
}

