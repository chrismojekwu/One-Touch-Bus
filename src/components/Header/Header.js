import React from 'react'
import logo from '../../img/OTB.png'

export default function Header(){
    return (
        <img className="logo" src={logo} alt="One Touch Bus"/>
    )
}

