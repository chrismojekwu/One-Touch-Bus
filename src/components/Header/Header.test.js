import React from "react";
import { render, screen } from "@testing-library/react";
import renderer from 'react-test-renderer';
import Header from './Header'

it('renders the header image without crashing', () => {
    render(<Header/>);
    expect(screen.getAllByRole('img').length).toBe(1);
})

it('renders the snapshot correctly', () => {
    const tree = 
    renderer.create(<Header/>).toJSON();
    expect(tree).toMatchSnapshot();
});