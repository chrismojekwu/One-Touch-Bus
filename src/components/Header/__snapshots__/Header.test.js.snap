// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot correctly 1`] = `
<img
  alt="One Touch Bus"
  className="logo"
  src="OTB.png"
/>
`;
